data/sql/updates/pending_db_world/rev_1570916858306186295.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1570916858306186295');
+
+DELETE FROM `trinity_string` WHERE `entry` = 30080;
+INSERT INTO `trinity_string` (`entry`, `content_default`, `content_loc1`, `content_loc2`, `content_loc3`, `content_loc4`, `content_loc5`, `content_loc6`, `content_loc7`, `content_loc8`)
+VALUES
+(30080,'The file ''opcode.txt'' is missing in the server working directory.',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL);
```

Comment:
For rus lang if need
```sql
(30080,'The file ''opcode.txt'' is missing in the server working directory.',NULL,NULL,NULL,NULL,NULL,NULL,NULL, 'Файл ''opcode.txt'' не найден в рабочей директории сервера');
```

Reply:
Could you perhaps add this and push a new commit?

Reply:
Nevermind, pushed a new commit adding the translation.